config/application.rb

```diff
@@ -350,8 +350,6 @@ class Application < Rails::Application
       ::Foreman::Plugin.all.select { |p| p.gettext_domain }.each do |plugin|
         domain = plugin.gettext_domain
         Foreman::Gettext::Support.add_text_domain domain, plugin.locale_path
```

Comment:
You could consider making this even shorter
```suggestion
        Foreman::Gettext::Support.add_text_domain plugin.gettext_domain, plugin.locale_path
```